Ginger/GingerAutoPilot/APIModelLib/SwaggerApi/OpenApiVer3.cs

```diff
@@ -124,7 +131,50 @@ public ObservableList<ApplicationAPIModel> OpenApiThree(SwaggerDocument Swaggerd
             return SwaggerModels;
         }
 
-        
+        public static Dictionary<string, string> GetExamplesFromOpenApiComponents(OpenApiComponents apiComponents)
+        {
+
+            //ObservableList<OptionalValue> exampleValues = new ObservableList<OptionalValue>();
+            Dictionary<string, string> exampleValues = new Dictionary<string, string>();
+            try
+            {
+                if (apiComponents.Schemas != null && apiComponents.Schemas.Count != 0)
+                {
+                    foreach (var schemaEntry in apiComponents.Schemas)
+                    {
+                        string schemaName = schemaEntry.Key;
+                        var schemaDefinition = schemaEntry.Value;
+
+                        if (schemaDefinition.ActualProperties != null && schemaDefinition.ActualProperties.Count > 0)
+                        {
+                            foreach (var item in schemaDefinition.ActualProperties)
+                            {
+                                var actualName = item.Key;
+                                var actualDefinition = item.Value.Example?.ToString();
+                                if (actualDefinition != null && !exampleValues.ContainsKey(actualName.ToLower()))
+                                {
+                                    exampleValues.Add(actualName.ToLower(), actualDefinition);
+                                }
+
+                            }
+                        }
+                        else if (schemaDefinition.Example != null)
+                        {
+                            if (!exampleValues.ContainsKey(schemaName.ToLower()))
+                            {
+                                exampleValues.Add(schemaName.ToLower(), schemaDefinition.Example.ToString());
+                            }
+                        }
+                    }
+                }
+            }
+            catch (Exception ex)
+            {
+                Reporter.ToLog(eLogLevel.INFO, "Example value was null in one of the tag", ex);
```

Comment:
Change loglevel to Error instead of Info as we caught a exception in catch block. If this is expected expception, please add a validation in try block

Reply:
Changed to error.